.github/workflows/commit-format.yml

```diff
@@ -108,3 +108,33 @@ jobs:
           done < <(git rev-list --reverse "$rev_list" )
 
           exit $retval;
+
+  check-issue-reference:
+    runs-on: ubuntu-latest
+    continue-on-error: true # We do not want to block merge if it is a legitimate GCC bugzilla reference.
+    name: check-issue-reference
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 0
+
+      - name: Check for issue number reference in commit messages
+        run: |
+          retval=0;
+          rev_list="origin/${{ github.event.pull_request.base.ref }}..${{ github.event.pull_request.head.sha }}"
+          for commit in $(git rev-list --reverse "$rev_list"); do
+            if [ "$(git show "$commit" | grep '#[0-9]*' | grep -v -i 'Rust GCC#[0-9]*' | wc -l)" -ne 0 ]; then
```

Comment:
```suggestion
            if [ "$(git show "$commit" | grep '#[0-9]*' | grep -v -i 'Rust-GCC/gccrs#[0-9]*' | wc -l)" -ne 0 ]; then
```

Reply:
Should we enforce the case too ?

Reply:
I don't think so, both are still matched by github. Rust-GCC/gccrs#3213 rust-gcc/GcCrS#3213